src/levanter/infra/ray_tpu.py

```diff
@@ -497,14 +585,13 @@ def run_on_pod_ray(
 
     This function is a Ray remote function that can be called from anywhere in the Ray cluster.
     """
-    if num_slices <= 0:
-        raise ValueError("num_slices must be greater than 0")
+    if not num_slices:
```

Comment:
probably doesn't matter but this doesn't check non-negativity anymore

Reply:
or actually I changed this to a more correct validation function